### ISSUE TITLE
Problem with swipe in Refresh layout  

### ISSUE THREAD

**Issue by mrx10** [android-cjj/Android-MaterialRefreshLayout#5]
when make small swipe (not full drag) Material refresh layout freezes  , progress doesn't rotate and on refresh not called 
as in link 

https://cloud.githubusercontent.com/assets/1844292/9881023/7d988ae4-5bce-11e5-9720-574ebfa0b4b4.png

**Comment by RonghuaDai** [comment]
me too

**Comment by amirarcane** [comment]
yes, it should get disappeared on small swipe but it freezes. not cool.
